tests/cases/compiler/exportDefaultEnumTargetES2015ModuleAMD.ts

```diff
@@ -0,0 +1,62 @@
+// https://github.com/Microsoft/TypeScript/issues/3792
+// @target: es2015

```

Comment:
Our compiler runner supports test variations for the same source. Instead of multiple `nnnTargetXModuleY.ts` files you can do this instead:
```suggestion
// @target: es2015, es5
```

Reply:
That would be nice, but it doesn't look like it's in yet?

` Error: Unknown value 'es5, es2015' for compiler option 'target'.`